packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/template.tsx

```diff
@@ -23,6 +23,12 @@ const Header = (props: OwnProps) => {
     props.sessionActions.logout()
   }, [])
 
+  const fabCallback = useCallback(() => {
+    props.modalActions.showModal('TRADE_MODAL', {
```

Comment:
(minor) - you can import ModalName and use it here like `ModalName.TRADE_MODAL` rather then just a pure string